examples/02_consul/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"log"
+
+	"github.com/hashicorp/consul/api"
+
+	"github.com/taxibeat/harvester/config"
+	"github.com/taxibeat/harvester/seed"
+	"github.com/taxibeat/harvester/seed/consul"
+)
+
+type fields struct {
+	consulParam *seed.Param
+}
+
+// ConfigAttrs sample config struct
+type ConfigAttrs struct {
+	Name      string `seed:"John Doe"`
+	Age       int64  `seed:"18" env:"ENV_AGE"`
+	ConsulVar string `seed:"b" env:"ENV_CONSUL_VAR" consul:"/harvester/example_02/consul_var"`
+}
+
+func main() {
+	consulConfig := api.DefaultConfig()
+	consulClient, err := api.NewClient(consulConfig)
+	if err != nil {
+		panic(err)
+	}
+	// Get a handle to the KV API
+	kv := consulClient.KV()
+
+	// PUT a new KV pair
+	p := &api.KVPair{Key: "harvester/example_02/consul_var", Value: []byte("bar")}
+	_, err = kv.Put(p, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	attrs := ConfigAttrs{}
+
+	cfg, err := config.New(&attrs)
```

Comment:
You should use harvester and not the individual components for config and seed.

Reply:
👍